modified minesweeper.py

Tounsi17Ghassen opened the pull request:
changed mines placing so the first click would be safe

Tounsi17Ghassen committed:
modified minesweeper.py

changed mines placing so the first click would be safe

Tounsi17Ghassen committed:
fixed number of mines and same coordinate mistakes

utkarsh13 reviewed (approved):
Changed mines placement so that the first click would be safe.

Tounsi17Ghassen committed:
added 2 new features , fixed 2 minor bugs

FEATURES :
1- when all linked mines are flagged , player can reveal all the cases next to that button by clicking it again
2- when all the remaining cases are mines , the game ends automatically as a win
BUGS
1- player cannot click on a flagged button
2- fixed a mistake on the random function (last case is number 80 not 81)

Tounsi17Ghassen committed:
added 2 new features , fixed 2 minor bugs

FEATURES :
1- when all linked mines are flagged , player can reveal all the cases next to that button by clicking it again
2- when all the remaining cases are mines , the game ends automatically as a win
BUGS :
1- player cannot click on a flagged button
2- fixed a mistake on the random function (last case is number 80 not 81)

Tounsi17Ghassen committed:
added statistics

Tounsi17Ghassen committed:
added hist.txt
